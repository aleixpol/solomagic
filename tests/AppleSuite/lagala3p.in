\ref something0
\mt somethingsomething lagalandi somethingsomething

\ref something1
\mt somethingsomething landing somethingsomething

