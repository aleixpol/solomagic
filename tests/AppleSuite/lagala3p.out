\ref something0
\mt somethingsomething lagalandi somethingsomething
\ma somethingsomething lagalangri somethingsomething

\ref something1
\mt somethingsomething landing somethingsomething
\ma somethingsomething landing somethingsomething
